python/pyarrow/tests/test_extension_type.py

```diff
@@ -1472,10 +1618,24 @@ def test_tensor_type_is_picklable(pickle_module):
         'fixed_shape_tensor[value_type=int64, shape=[2,2,3], dim_names=[C,H,W]]'
     )
 ])
-def test_tensor_type_str(tensor_type, text):
+def test_tensor_type_str(tensor_type, text, pickle_module):
```

Comment:
Can you write a separate test for pickling instead of adding to this function?

Reply:
Done.